### DISCUSSION TITLE
blit모바일에서 안되나요?

### DISCUSSION THREAD

**Discussion by jiwoo50** [naverz/zepeto-studio-kor#985] [5. Q&A]
어떤 영역에 닿으면 화면이 흑백으로 나오도록 grayscale쉐이더와 Graphics.Blit을 이용해 구현했는데, Graphics.Blit을 유니티에서 빌드할때에는 문제가 없었는데 Graphics.Blit를 모바일 환경에서 사용하니 화면이 검은색으로 나오고 다시 해당 스크립트를 삭제하니 정상적으로 출력되었습니다. Graphics.Blit아직 모바일에서 잘 작동이 안되나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, Graphics.Blit는 사용 가능하며 DJ Neon Party등의 공식 월드에 사용되었습니다. 
참고 부탁 드립니다 :) https://github.com/naverz/zepeto-studio-kor/discussions/216#discussioncomment-2042964
